doc/releases/release-notes-3.2.rst

```diff
@@ -154,6 +157,11 @@ Deprecated in this release
   Its removal is scheduled for the 3.4 release. Refer to :ref:`pinctrl-guide`
   for more details on pin control.
 
+* :c:macro:`FLASH_AREA_LABEL_EXISTS`, :c:macro:`FLASH_AREA_ID`,
```

Comment:
@de-nordic @carlescufi Can you confirm that it is true.

Reply:
ref.: https://github.com/zephyrproject-rtos/zephyr/pull/48174

Reply:
actually this PR should have been merged: https://github.com/zephyrproject-rtos/zephyr/pull/49974